projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -18,15 +19,17 @@ import { CartItemComponentOptions } from '../cart-item/cart-item.component';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CartItemListComponent {
-  @Input() readonly = false;
+  @Input() readonly: boolean = false;
 
-  @Input() hasHeader = true;
+  @Input() hasHeader: boolean = true;
 
   @Input() options: CartItemComponentOptions = {
     isSaveForLater: false,
     optionalBtn: null,
   };
 
+  @Input() cart: { userId: string; cartId: string };
```

Comment:
Can we add `userIdService` to constructor and change this input from `cart` -> `cartId` and when we need to pass user id use the value from the userIdService?

Reply:
proposition in PR https://github.com/SAP/spartacus/pull/11642